terraform/account/elasticache.tf

```
@@ -0,0 +1,24 @@
+resource "aws_elasticache_subnet_group" "brute_force_cache_subnet" {
```

I think this could be called "private". It can be used on any number of elasticache clusters

```suggestion
resource "aws_elasticache_subnet_group" "private_subnets" {
```